client/src/Home.scss

```diff
@@ -0,0 +1,22 @@
+.home__top {
```

Comment:
Look at how many FEWER classes I needed to define for the Home page if I simpy use Bootstrap grid classes to do layout.